go-controller/pkg/ovn/egressfirewall.go

```diff
@@ -0,0 +1,304 @@
+package ovn
+
+import (
+	"fmt"
+	"net"
+	"reflect"
+	"strings"
+	"sync"
+
+	"github.com/ovn-org/ovn-kubernetes/go-controller/pkg/util"
+
+	egressfirewallapi "github.com/ovn-org/ovn-kubernetes/go-controller/pkg/crd/egressfirewall/v1"
+
+	"k8s.io/klog"
+)
+
+const (
+	defaultStartPriority = 2000
+)
+
+type egressFirewall struct {
+	sync.Mutex  //not sure if needed
```

Comment:
It's definitely not needed if you never use it. What was your idea here?

Reply:
done